test/Atmos/Model/driver_spacing_test.jl

```diff
@@ -0,0 +1,40 @@
+using MPI
+using ArgParse
+using ClimateMachine
+ClimateMachine.init()
+using ClimateMachine.MPIStateArrays
+using ClimateMachine.Atmos
+using ClimateMachine.ConfigTypes
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.DGMethods
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.Mesh.Geometry
+using ClimateMachine.Orientations
+
+using Test
+
+@testset "EmptyModel Get Nodal Distance GCM" begin
+    FT = Float64
+    Npoly = (5, 5)
+    Nelem = (8, 5)
+    Δh, Δv = get_grid_spacing_gcm(Npoly, Nelem, FT(30e3))
+    # Values established from "complete" driver run 
+    # min_node_distance is independently tested.
+    @test Δh ≈ 103918.2452822826
+    @test Δv ≈ 704.834028207901
+end
+@testset "EmptyModel Get Nodal Distance LES" begin
```

Comment:
Same question as  on line 16 above. Should this be printing `"EmptyModel Get Element Size LES"` or `"EmptyModel Get Grid Size LES"`? (again, unless I am completely misunderstanding)